app/controllers/admin/api/api_docs_services_controller.rb

```diff
@@ -132,8 +133,20 @@ def api_docs_params(*extra_params)
     params.require(:api_docs_service).permit(*permit_params)
   end
 
+  def api_docs_services
+    collection = current_account.api_docs_services.accessible
+    return collection unless current_user
+    collection.permitted_for(current_user)
+  end
+
   def find_api_docs_service
-    @api_docs_service = current_account.api_docs_services.find(params[:id])
+    @api_docs_service = api_docs_services.find(params[:id])
```

Comment:
Perhaps we should also add an `attr_reader :api_docs_service` and then in all the other methods (such as `update`) we could use `api_docs_service` instead of `@api_docs_service`.
Not a big deal either way 👍